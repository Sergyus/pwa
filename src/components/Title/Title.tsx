import styles from './styles.module.scss';
import React from 'react';

type Props = {
  title: string;
};

export default function Title(props: Props): JSX.Element {
  return <h1 className={styles.title}>{props.title}</h1>;
}
