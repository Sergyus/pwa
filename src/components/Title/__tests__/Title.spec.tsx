import React from 'react';
import { render, screen } from '@testing-library/react';
import Title from '@components/Title';

describe('<Title />', () => {
  it('should render with children props', () => {
    const title = 'Test title';
    const { asFragment } = render(<Title title={title} />);
    expect(screen.getByRole('heading', { level: 1 })).toBeInTheDocument();
    expect(asFragment()).toMatchSnapshot();
  });
});
