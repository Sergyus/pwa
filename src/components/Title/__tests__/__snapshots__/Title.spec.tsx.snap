// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render with children props 1`] = `
<DocumentFragment>
  <h1
    class="title"
  >
    Test title
  </h1>
</DocumentFragment>
`;
